docs/src/language_servers.md

```diff
@@ -542,6 +542,34 @@ There are multiple options:
     }
     ```
 
+### Steep
+
+1. Add the `steep` gem (see [github:soutaro/steep](https://github.com/soutaro/steep)) into your Gemfile and install it
+
+    ```sh
+    bundle install
```

Comment:
I don't know the ruby ecosystem but why are those instructions say to add to "gemfile" while instructions for other servers say to use `gem install`? Is it because this approach is local to the project while `gem install` is for installing globally? If so then shouldn't we recommend to install globally in this case also?

Reply:
I would say that project-specific setup should be combined with a project-specific configuration. Unless there is something specific about this server that makes it more suitable for project-specific use, I'd suggest following other setups for global server.